lib/smart_proxy_reports/ansible_processor.rb

```diff
@@ -76,16 +75,43 @@ def spool_report
       payload = measure :format do
         report_hash.to_json
       end
-      SpooledHttpClient.instance.spool(report_id, payload)
+      SpooledHttpClient.instance.spool("report", report_id, payload)
     end
 
-    private
+    def find_ansible(array)
+      @data["results"]&.each do |result|
+        unless result["result"].nil? && result["result"]["ansible_facts"].nill?
```

Comment:
Double l in nill

Reply:
Could be named `find_facts_task` or something like that.